### PR TITLE
reconfigure `html-webpack-plugin` and fix dependencies to fix i18next

### PR DESCRIPTION
**Pull request by merobi-hub**
Signed-off-by: Michael Robinson <merobi@gmail.com>

### Problem

The UI was breaking after i18next was added to the project due to the configuration of `html-webpack-plugin`.
 
### Solution

This change sets the `inject` option to `false` to avoid duplicate instances of React and React DOM, which were causing a hard-to-parse invalid hooks error. Now, the UI with i18next appears to be working as expected.

### Checklist

- [x] You've [signed-off](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#sign-your-work) your work
- [ ] Your changes are accompanied by tests (_if relevant_)
- [x] Your change contains a [small diff](https://kurtisnusbaum.medium.com/stacked-diffs-keeping-phabricator-diffs-small-d9964f4dcfa6) and is self-contained
- [ ] You've updated any relevant documentation (_if relevant_)
- [ ] You've updated the [`CHANGELOG.md`](https://github.com/MarquezProject/marquez/blob/main/CHANGELOG.md#unreleased) with details about your change under the "Unreleased" section (_if relevant, depending on the change, this may not be necessary_)
- [ ] You've versioned your `.sql` database schema migration according to [Flyway's naming convention](https://flywaydb.org/documentation/concepts/migrations#naming) (_if relevant_)
- [ ] You've included a [header](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#copyright--license) in any source code files (_if relevant_)

### CODE REVIEW

**Comment by merobi-hub** [comment]
Caps of the UI after running `./docker/up.sh --build --seed`:
Screen Shot 2022-12-12 at 16 11 37
Screen Shot 2022-12-12 at 16 12 33

File: web/src/i18n/config.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import { initReactI18next } from 'react-i18next'
 import LanguageDetector from 'i18next-browser-languagedetector'
+// import i18next from 'i18next';
```
Comment (wslulciuc) [inline web/src/i18n/config.ts:3]: _Minor_: should we remove the import instead of commenting it out?

File: web/src/i18n/config.ts [hunk 2]
```diff
@@ -425,5 +427,13 @@ i18next
         }
       }
     },
+    defaultNS,
     detection: DETECTION_OPTIONS
   })
+
+// export default i18n
```
Comment (wslulciuc) [inline web/src/i18n/config.ts:434]: _Minor_: similar comment as in https://github.com/MarquezProject/marquez/pull/2315/files#r1046400526

**Review by wslulciuc — COMMENTED** [review]
Left a couple of comments, but otherwise LGTM but will have @phixMe make the official approval.

**Review by phixMe — APPROVED** [review]
Thanks @merobi-hub

**Pull request merged**